parsec/core/cli/stats_organization.py

```diff
@@ -41,3 +72,42 @@ def stats_organization(
 ) -> None:
     with cli_exception_handler(debug):
         trio_run(_stats_organization, organization_id, addr, administration_token)
+
+
+@click.command(short_help="Get a per-organization report of server usage within a period of time")
+@click.option("--addr", "-B", required=True, type=BackendAddr.from_url, envvar="PARSEC_ADDR")
+@click.option(
+    "--admin-token",
+    "-T",
+    required=True,
+    envvar="PARSEC_ADMINISTRATION_TOKEN",
+    help="Passing the admin token as an argument represents a security risk, prefer using the environment variable for that.",
+)
+@click.option(
+    "--date-from",
+    type=click.UNPROCESSED,
+    callback=_validate_date,
+    help="A RFC 3339 compliant timestamp eg. (2020-12-09 16:09:53+00:00).",
+)
+@click.option(
+    "--date-to",
+    default=DateTime.now().to_rfc3339(),
+    type=click.UNPROCESSED,
+    callback=_validate_date,
+    help="A RFC 3339 compliant timestamp eg. (2020-12-09 16:09:53+00:00)",
+)
+@click.option("--output", type=str)
+@click.option("--format", default="json", type=click.Choice(["json", "csv"]))
+@cli_command_base_options
+def stats_server(
+    addr: BackendAddr,
+    admin_token: str,
+    date_from: DateTime,
```

Comment:
Since `date-from` is optional, won't this need to be set to `Optional` ?

Reply:
Yes, I've updated the PR